tests/e2e/service_invocation/service_invocation_test.go

```diff
@@ -319,113 +321,120 @@ var grpcProxyTests = []struct {
 }
 
 func TestServiceInvocation(t *testing.T) {
```

Comment:
Reviewers: the reason why this PR appears to be so large is because these code blocks were indented more :) However not much has changed inside them, besides replacing `"serviceinvocation-caller"` with `targetApp`